<launch>
	<group ns="voice_recognizer_pocketsphinx">
		<rosparam command="load" file="$(find voice_recognizer_pocketsphinx)/config/voice_recognizer_pocketsphinx_kws.yaml" ns="kws_recognizer"/>		

		<node name="kws_recognizer" pkg="voice_recognizer_pocketsphinx" type="kws_recognizer.py" output="screen" >
    	</node>
	</group>
</launch>